scripts/setup/sha256-tarball-to

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -e
+set -x
+
+SHA256="$1"
```

Comment:
Is it worth converting this to use shell getopt and/or using more variables just to be a bit more readable, since we're now makign this a generic script?

The `while` loop below probably needs a comment, at least.